src/main/frontend/handler.cljs

```diff
@@ -73,51 +73,48 @@
     (state/pub-event! [:instrument {:type :blocks/count
                                     :payload {:total (db/blocks-count)}}])))
 
-(defn store-schema!
-  []
-  (storage/set :db-schema (assoc db-schema/schema
-                                 :db/version db-schema/version)))
-
 (defn restore-and-setup!
-  [repos old-db-schema]
```

Comment:
Better not to change the original logic here. This breaks mobile loading.

Some fixes are merged recently to fix mobile loading and restoring.
#6561 https://github.com/logseq/logseq/commit/63d268f5f00df87b92b39e1bebc8b6274a7d0492